BitmarkRegistry/AppDelegate.swift

```diff
@@ -74,18 +74,28 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
 }
 
 private extension AppDelegate {
+
+  /**
+   1. evaluate Touch/Face Id
+   2. request mobile_server_url to get and store jwt
+   */
   func evaluatePolicyWhenUserSetEnable() {
     guard Global.currentAccount != nil else { return }
-    guard UserSetting.shared.getTouchFaceIdSetting() else { return }
+    guard UserSetting.shared.getTouchFaceIdSetting() else {
+      AccountService.requestJWT(account: Global.currentAccount!)
```

Comment:
Try to avoid force cast as much as possible, although the logic is correct.
I'd suggest to assign account on `guard Global.currentAccount != nil else { return }` and use it here.